src/test/java/uk/gov/hmcts/reform/sscs/util/SscsOcrDataUtilTest.java

```diff
@@ -226,4 +226,15 @@ public void givenAnOcrDateWithInvalidFormat_thenAddError() {
 
         assertEquals("hearingDate is an invalid date field. Needs to be in the format dd/mm/yyyy", errors.get(0));
     }
+
+    @Test
+    public void givenAnOcrInvalidDate_thenAddError() {
+        pairs.put("hearingDate", "29/02/2018");
+
+        List<String> errors = new ArrayList<>();
+
+        generateDateForCcd(pairs, errors, "hearingDate");
+
+        assertEquals("hearingDate is an invalid date field. Needs to be in the format dd/mm/yyyy", errors.get(0));
```

Comment:
>Needs to be in the format dd/mm/yyyy
It is in this format though

Reply:
True, I can maybe change the error message